scipy/spatial/_spherical_voronoi.py

```diff
@@ -108,6 +126,14 @@ class SphericalVoronoi:
         Threshold for detecting duplicate points and
         mismatches between points and sphere parameters.
         (Default: 1e-06)
+    method : string
+        The method to determine the vertices of the Voronoi regions.
+        Must be one of
+        - `circumcenters`, `circumcenter`, or `cc`, for tetrahedron
+          circumcenters or 
+        - `centroids`, `centroid`, or `centr` for cetroids of the 
```

Comment:
```suggestion
        - `centroids`, `centroid`, or `centr` for centroids of the 
```

Reply:
I'm not sure it is helpful to have this many different ways to refer to the method choices.